deploy/kubernetes/rbac.yaml

```diff
@@ -51,6 +51,14 @@ rules:
     resources: ["customresourcedefinitions"]
     verbs: ["create", "list", "watch", "delete", "get", "update"]
 
+#Secret permission is optional.
+#Enable it if you need value from secret.
```

Comment:
s/if you need value from secret/if your driver needs secret

Reply:
fixed